duckduckgo/BrowserServicesKit
# autofill update

Task/Issue URL: https://app.asana.com/0/392891325557410/1200275136285356
Tech Design URL:
CC:

**Description**:

Bring autofill support up to date.

**Steps to test this PR**:
1. Check out test branch and ensure that both the user's address and alias are shown when signing up to mailing lists etc



**OS Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
